Packs/Exabeam/Integrations/Exabeam/README.md

```diff
@@ -1,288 +1,60 @@
-<p>
 The Exabeam Security Management Platform provides end-to-end detection, User Event Behavioral Analytics, and SOAR.
```

Comment:
Is this up to date with the changes in the integration setup window?
Also, some of your outputs are quite long - shorten them manually so it will be easier to follow.